### PR TITLE
Random

### LINKED ISSUE #36: Add random() wrapper
Slides team uses this in Session 3 for a pong exercise to start the ball in a random direction. It's not mandatory to have but would be nice

Acceptance Criteria:
* [x] Add `random()` that returns a float r, `0 <= r <= 1.0`
* [x] Add `randint(n)` that returns an int r, `0 <= r < n `
* [x] Add to documentation

### PR DESCRIPTION
**Pull request by Descent098**
- [x] Add random() that returns a float r, 0 <= r <= 1.0
 - [x]  Add random(n) that returns an int r, 0 <= r < n (not sure if we should use < n or <= n)
 - [x]  Add to documentation (Added to #35 )

closes #36

### CODE REVIEW

**Comment by Descent098** [comment]
Waiting on #35

File: spark/core.py [hunk 1]
```diff
@@ -512,4 +514,19 @@ def check_coords(self, func_name, *args, width_only=False):
     def arc_args(self, *args):
         return (args[0], args[1], args[2] / 2, 0, 2 * pi)
 
+    # Global namespace alias of random.random()
+    def random(self, *args):
+        argc = len(args)
+        if argc != 0:
+            raise ArgumentNumError("Random", 0, argc)
```
Comment (AlphaRLee) [inline spark/core.py]: Should be lowercase to match the function name
```suggestion
            raise ArgumentNumError("random", 0, argc)
```

File: spark/core.py [hunk 2]
```diff
@@ -512,4 +514,19 @@ def check_coords(self, func_name, *args, width_only=False):
...
+        return random.random()
+
+    # Global namespace alias of random.randint()
+    def randint(self, *args):
+        argc = len(args)
+        
+        if argc != 1:
+            raise ArgumentNumError("Randint", 1, argc)
```
Comment (AlphaRLee) [inline spark/core.py]: See comment above
```suggestion
            raise ArgumentNumError("randint", 1, argc)
```

Comment (Descent098) [reply spark/core.py]: resolved in latest commit

File: spark/core.py (hunk 1 above)
Comment (Descent098) [reply spark/core.py]: Resolved in latest commit

**Review by lawruble13 — APPROVED** [review]
Changes look good, I added a fairly simple test that starts a bouncing ball in a random direction.

**Pull request merged**